WooCommerce/Classes/POS/ViewModels/ItemListViewModel.swift

```diff
@@ -12,6 +12,8 @@ final class ItemListViewModel: ItemListViewModelProtocol {
 
     @Published private var currentPage: Int = Constants.initialPage
 
+    private(set) var shouldShowGhostableItemCard: Bool?
```

Comment:
nitpick; no need to have boolean optional

```suggestion
    private(set) var shouldShowGhostableItemCard: Bool = false
```

Reply:
True! Updated on https://github.com/woocommerce/woocommerce-ios/pull/14373/commits/3b334d0aef5c8353a16ac1034a0225ff56e7e307